test/unit/test_defmodel

```diff
@@ -0,0 +1,210 @@
+#! /bin/bash
```

Comment:
this file should not be committed. You may need to add it to .gitignore

Reply:
:man_facepalming:, yep. Will need to add that to the `.gitignore`. Glad you caught that :+1: